src/labone/core/subscription.py

```diff
@@ -173,6 +173,8 @@ def put_nowait(self, item: AnnotatedValue) -> None:
             raise errors.StreamingError(msg)
         return super().put_nowait(item)
 
+    # def put_streaming_error(self, err: hpk_schema.Error) -> None:
```

Comment:
Left for future implementation, or forgotten?

Reply:
🙈 forgotten piece of code ... I want to add the stream error support in the future but it would be to much for this MR 😇